kalohq/ui
# Audit dependencies and upgrade Sinon

This PR does a few things: upgrades dependencies that have security warnings against them, upgrades `sinon` to latest and improves how `sinon` is used.

#### How to verify

1. Check out this branch
1. `npm ci` (or `npm install` if you're using old school `npm`)
1. `npm test`
